soc/xtensa/nxp_adsp/imx8ulp/include/soc/memory.h

```diff
@@ -0,0 +1,212 @@
+#define SAI_5_SIZE	0x00010000
+#define SAI_6_BASE	0x2DA90000
+#define SAI_6_SIZE	0x00010000
+#define SAI_7_BASE	0x2DAA0000
+#define SAI_7_SIZE	0x00010000
+
+#define UUID_ENTRY_ELF_BASE	0x1FFFA000
+#define UUID_ENTRY_ELF_SIZE	0x6000
+
+#define LOG_ENTRY_ELF_BASE	0x20000000
+#define LOG_ENTRY_ELF_SIZE	0x2000000
+
+#define EXT_MANIFEST_ELF_BASE	(LOG_ENTRY_ELF_BASE + LOG_ENTRY_ELF_SIZE)
+#define EXT_MANIFEST_ELF_SIZE	0x2000000
+
+/*
+ * The Heap and Stack on i.MX8 are organized like this :-
+ *
+ * +--------------------------------------------------------------------------+
+ * | Offset              | Region         |  Size                             |
+ * +---------------------+----------------+-----------------------------------+
+ * | SDRAM_BASE          | RO Data        |  SOF_DATA_SIZE                    |
+ * |                     | Data           |                                   |
+ * |                     | BSS            |                                   |
+ * +---------------------+----------------+-----------------------------------+
+ * | HEAP_SYSTEM_BASE    | System Heap    |  HEAP_SYSTEM_SIZE                 |
+ * +---------------------+----------------+-----------------------------------+
+ * | HEAP_RUNTIME_BASE   | Runtime Heap   |  HEAP_RUNTIME_SIZE                |
+ * +---------------------+----------------+-----------------------------------+
+ * | HEAP_BUFFER_BASE    | Module Buffers |  HEAP_BUFFER_SIZE                 |
+ * +---------------------+----------------+-----------------------------------+
+ * | SOF_STACK_END       | Stack          |  SOF_STACK_SIZE                   |
+ * +---------------------+----------------+-----------------------------------+
+ * | SOF_STACK_BASE      |                |                                   |
+ * +---------------------+----------------+-----------------------------------+
+ */
+
+#define SRAM_OUTBOX_BASE	SDRAM1_BASE
+#define SRAM_OUTBOX_SIZE	0x1000
+#define SRAM_OUTBOX_OFFSET	0
+
+#define SRAM_INBOX_BASE		(SRAM_OUTBOX_BASE + SRAM_OUTBOX_SIZE)
+#define SRAM_INBOX_SIZE		0x1000
+#define SRAM_INBOX_OFFSET	SRAM_OUTBOX_SIZE
+
+#define SRAM_DEBUG_BASE		(SRAM_INBOX_BASE + SRAM_INBOX_SIZE)
+#define SRAM_DEBUG_SIZE		0x2800
+#define SRAM_DEBUG_OFFSET	(SRAM_INBOX_OFFSET + SRAM_INBOX_SIZE)
+
+#define SRAM_EXCEPT_BASE	(SRAM_DEBUG_BASE + SRAM_DEBUG_SIZE)
+#define SRAM_EXCEPT_SIZE	0x800
+#define SRAM_EXCEPT_OFFSET	(SRAM_DEBUG_OFFSET + SRAM_DEBUG_SIZE)
+
+#define SRAM_STREAM_BASE	(SRAM_EXCEPT_BASE + SRAM_EXCEPT_SIZE)
+#define SRAM_STREAM_SIZE	0x1000
+#define SRAM_STREAM_OFFSET	(SRAM_EXCEPT_OFFSET + SRAM_EXCEPT_SIZE)
+
+#define SRAM_TRACE_BASE		(SRAM_STREAM_BASE + SRAM_STREAM_SIZE)
+#define SRAM_TRACE_SIZE		0x1000
+#define SRAM_TRACE_OFFSET (SRAM_STREAM_OFFSET + SRAM_STREAM_SIZE)
+
+#define SOF_MAILBOX_SIZE	(SRAM_INBOX_SIZE + SRAM_OUTBOX_SIZE \
+				+ SRAM_DEBUG_SIZE + SRAM_EXCEPT_SIZE \
+				+ SRAM_STREAM_SIZE + SRAM_TRACE_SIZE)
+
+/* Heap section sizes for module pool */
```

Comment:
These are nowhere used - the HEAP_* and SOF_STACK*

Reply:
Don't add all of this unnecessary information unless you know you're going to be using it in the future. Please remove it if not.